core/src/main/python/wlsdeploy/tool/util/wlst_helper.py

```diff
@@ -1361,6 +1361,45 @@ def server_config(self):
             raise pwe
         self.__logger.exiting(class_name=self.__class_name, method_name=_method_name)
 
+    def get_domain_name_online(self):
+        """
+        Get the domain name online.
+        :raises Exception for the specified tool type: if a WLST error occurs
+        """
+        _method_name = 'get_domain_name_online'
+        self.__logger.entering(class_name=self.__class_name, method_name=_method_name)
+        try:
+            self.server_config()
+            self.cd('/')
+            name = self.get('Name')
+        except self.__load_global('WLSTException'), e:
+            pwe = exception_helper.create_exception(self.__exception_type, 'WLSDPLY-00065',
+                                                    _format_exception(e), error=e)
+            self.__logger.throwing(class_name=self.__class_name, method_name=_method_name, error=pwe)
+            raise pwe
+        return name
+        self.__logger.exiting(class_name=self.__class_name, method_name=_method_name)
```

Comment:
Should this logging call be before the return?

Reply:
Fixed